sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/PartitionPump.cs

```diff
@@ -16,7 +16,7 @@ namespace Azure.Messaging.EventHubs.Processor
     ///
     internal class PartitionPump
     {
-        /// <summary>The <see cref="EventHubRetryPolicy" /> used to verify whether an exception is retryable or not.</summary>
+        /// <summary>The <see cref="EventHubRetryPolicy" /> used to verify whether an exception is retriable or not.</summary>
```

Comment:
Technically, I think that both spellings are correct, but the rest of the codebase uses `retriable` so I let the spell checker have its way.